ckan/lib/helpers.py

```diff
@@ -933,8 +933,11 @@ def build_extra_admin_nav():
     admin_tabs_dict = config.get('ckan.admin_tabs')
     output = ''
     if admin_tabs_dict:
-        for key in admin_tabs_dict:
-            output += build_nav_icon(key, admin_tabs_dict[key])
+        for k, v in admin_tabs_dict.iteritems():
+            if v['icon']:
```

Comment:
Doesn't `icon` has `None` as default value, so you can just safetuly call  `build_nav_icon(k, v['label'], icon=v['icon'])` without any conditional clause?

Reply:
Thank you for review
I felt that I have to use them separately because there are two functions in the ckan core.
build_nav_icon function may change to a policy that will have a default image in the future.
I'm sorry I do not know the policy of the ckan team. :)
But, I think that if I do not set the icon, the icon should not be displayed.

Reply:
case 1) icon1(ckan_icon) exists in mapper, call `add_ckan_admin_tabs(..., icon=icon2) `
=> icon1 or icon2?

case 2) icon1(ckan_icon) exists in mapper, call `add_ckan_admin_tabs(..., icon=None) `
=> icon1 or just label without icon
-> If icon1 should be visible, then I should call the function as follows.

`if v['icon']:`
`    output += build_nav_icon(k, v['label'], icon=v['icon'])`
`else:`
`    output += build_nav_icon(k, v['label'])`


I would really appreciate your feedback

Reply:
Good point, you are absolutely right here, so I'm going to merge this PR